src/core/lv_obj.c

```diff
@@ -59,6 +63,13 @@ const lv_obj_class_t lv_obj_class = {
     .group_def = LV_OBJ_CLASS_GROUP_DEF_FALSE,
     .instance_size = (sizeof(lv_obj_t)),
     .base_class = NULL,
+#if LV_USE_OBJID_BUILTIN
+    .obj_count = &obj_count,
+#endif
+
+#if LV_USE_CLASS_NAME
```

Comment:
Should we remove config `LV_USE_CLASS_NAME`, make it alwasy enabled? It only consumes little flash size.

Reply:
I agree!